packages/plugin-eslint-rules/README.md

```diff
@@ -0,0 +1,61 @@
+# plugin-eslint-rules
+
+is-compatible is a simple eslint plugin that checks whether imports from any of the Grafana packages (`@grafana/ui`, `@grafana/data` and `@grafana/runtime`) from within a Grafana plugin source code exist in all the Grafana runtimes that the plugin is supposed to support (as specified in the `grafanaDependency` in `plugin.json`).
+
+![Demo](./images/plugin-eslint-rules.gif)
```

Comment:
We didn't copy this image across. We can remove it if we don't see the value. Alternatively if we do want to keep it we need it to be an absolute path so it shows up in the npmjs readme preview ([e.g.](https://www.npmjs.com/package/@grafana/create-plugin)).

Reply:
Would it be enough to copy the file to the root of the `plugin-eslint-rules` because I'm not sure what the absolute path means in the context of npmjs?

Reply:
Never mind, the image shows the incorrect package name in the animation so I'll just remove it.

Reply:
Fixed 👍